docs/medical-api/api-reference/document/list-documents.mdx

```diff
@@ -4,6 +4,15 @@ description: "Lists all Documents that can be retrieved for a Patient."
 api: "GET /medical/v1/document"
 ---
 
+This endpoint returns the document references associated with a Patient.
+
+Because this might contain a large number of records that have to obtained from HIEs,
+this endpoint works asynchronously:
+1. Upon request, it fires an asynchronous document query with HIEs and immediately returns the existing 
+document references stored at Metriport;
+   - When the asynchronous document query finishes, it stores new/updated document references for future requests;
+1. When requested again, it will return the existing documents and trigger a new document query (step 1).
```

Comment:
By this logic, will the `queryStatus` always be `processing`?

Reply:
Yes, until we find a different trigger for the doc query.

Actually, if the patient is missing CW info it won't query CW and the status will be `completed` - but this should never happen.